kodilla-collections/src/main/java/com/kodilla/collections/interfaces/homework/Ford.java

```diff
@@ -0,0 +1,30 @@
+package com.kodilla.collections.interfaces.homework;
+
+public class Ford implements Car {
+
+    private double speed = 150;
```

Comment:
a to juz bez sensu zrobiles.

private static final double ACCELERATION = 15;
private static final double DECELERATION = 20;

private double speed;